packages/react-components/src/organisms/TeamCreateOutputForm.tsx

```diff
@@ -0,0 +1,27 @@
+import { css } from '@emotion/react';
+import { Button } from '../atoms';
+import { perRem } from '../pixels';
+import { contentSidePaddingWithNavigation } from '../layout';
+
+const controlsContainerStyles = css({
+  display: 'grid',
+  gridAutoFlow: 'column',
+  alignItems: 'end',
+  justifyContent: 'end',
+  columnGap: `${12 / perRem}em`,
+  padding: `${36 / perRem}em ${contentSidePaddingWithNavigation(8)} ${
+    48 / perRem
+  }em `,
+});
+
+const TeamCreateOutputForm: React.FC<{ onCreate: () => void }> = ({
+  onCreate,
```

Comment:
Usually we default functions like this to noop (you can import it). This just means you don't neeed floating empty functions everywhere like your storybook etc.